api/src/test/resources/testAppDataDir/configuration/jsonkeyvalues/config.json

```diff
@@ -28,5 +28,11 @@
     "report.newEpisodesOfDiseases.referredCodedanswer.concept" : "d7f6f425-df28-47f4-9474-ba85a462a261",
     "report.newEpisodesOfDiseases.active" : "true",
     "report.vaccination.active" : "true",
-    "report.ecv.active" : "true"
+    "report.ecv.active" : "true",
+    "report.prenatalVisitType" : "35ba9aff-901c-49dc-8630-a59385480d18",
+    "report.vaccinations" : "3c25e650-ccfa-48da-b276-a7872796b113",
+    "report.vaccinationSequenceNumber" : "39741bfe-36a0-4a5b-9c64-fd4caaadca01",
+    "report.boosterSequenceNumber" : "1f4109b3-b201-46df-a341-dcfa5299a187",
+    "report.vaccinationList" : "9199a4c6-b15f-482c-aa0e-c839bd0cf535,0661812a-6ff7-42f9-bbe7-f79c5bdd0f58 1,0661812a-6ff7-42f9-bbe7-f79c5bdd0f58 2,0661812a-6ff7-42f9-bbe7-f79c5bdd0f58 3,30fd2e76-bfcb-49b4-8668-64fd3d3ec0ea,613c8c54-a1f5-435c-a1f0-ec9beaa42d06 1,613c8c54-a1f5-435c-a1f0-ec9beaa42d06 2,613c8c54-a1f5-435c-a1f0-ec9beaa42d06 3,9a9b3ca9-4ef9-4cb6-8a6d-ed9a7c32c46e 1,9a9b3ca9-4ef9-4cb6-8a6d-ed9a7c32c46e 2,5bc84919-86f1-4d9a-8b08-73cbb853b574 1,5bc84919-86f1-4d9a-8b08-73cbb853b574 2,3afaa3ce-0dbe-4ac8-964a-1913ee930d34 1,3afaa3ce-0dbe-4ac8-964a-1913ee930d34 2",
+    "report.ecvList" : "9199a4c6-b15f-482c-aa0e-c839bd0cf535,0661812a-6ff7-42f9-bbe7-f79c5bdd0f58 2,30fd2e76-bfcb-49b4-8668-64fd3d3ec0ea,613c8c54-a1f5-435c-a1f0-ec9beaa42d06 3,9a9b3ca9-4ef9-4cb6-8a6d-ed9a7c32c46e 2,5bc84919-86f1-4d9a-8b08-73cbb853b574 1"
```

Comment:
@reagan-meant, could you delimit these `[uuid|sequence-no]`s with a full-colon `:` instead rather than a space to minimise ambiguity? You'd also have to clearly document this construct.

Reply:
Done